python/cuml/CMakeLists.txt

```diff
@@ -85,11 +88,16 @@ if(NOT ${CUML_CPU})
 
     # Statically link dependencies if building wheels
     set(CUDA_STATIC_RUNTIME ON)
-    set(CUDA_STATIC_MATH_LIBRARIES ON)
     set(CUML_USE_RAFT_STATIC ON)
     set(CUML_USE_FAISS_STATIC ON)
     set(CUML_USE_TREELITE_STATIC ON)
     set(CUML_USE_CUMLPRIMS_MG_STATIC ON)
+    # Use the CUDA library wheels if possible
```

Comment:
This comment is underneath the previous comment `# Statically link dependencies ...`. Can we mark this as "shared" for clarity?
```suggestion
    # Link to the CUDA wheels with shared libraries for CUDA 12+
```